src/compilers/statement.cr

```diff
@@ -1,10 +1,18 @@
 module Mint
   class Compiler
-    def _compile(node : Ast::Statement) : String
+    def compile(node : Ast::Statement, last : Bool) : String
+      if checked.includes?(node)
+        _compile(node, last)
+      else
+        ""
+      end
```

Comment:
```suggestion
      node.in?(checked) ? _compile(node, last) : ""
```

Reply:
We have the same style in 9 places in the code, so I won't commit this, but if you like we can change all of them in a separate PR.

Reply:
I'd prefer changing all of the occurrences to the above.